docs/static/includes/module-features/bicepBadges.md

```diff
@@ -164,4 +164,5 @@
 | 162 | res/web<p>serverfarm | [![avm.res.web.serverfarm](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.serverfarm.yml/badge.svg?branch=main)](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.serverfarm.yml) |
 | 163 | res/web<p>site | [![avm.res.web.site](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.site.yml/badge.svg?branch=main)](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.site.yml) |
 | 164 | res/web<p>static-site | [![avm.res.web.static-site](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.static-site.yml/badge.svg?branch=main)](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.web.static-site.yml) |
+| 165 | utl/types<p>avm-common-types |  |
```

Comment:
Not yet published

Reply:
True - but I truth be told I find it more concerning that there's no pipeline badge, right? 😄

Reply:
Found the issue. Will push it into this branch :)

Reply:
done

Reply:
For me the issue was already at "Module", didn't even go to "Status" 😄